orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/service/WebhookService.groovy

```diff
@@ -30,6 +30,9 @@ import org.springframework.web.client.RestTemplate
 @Service
 class WebhookService {
 
+  // These headers create a security vulnerability.
+  static final List<String> headerBlacklist = ["x-spinnaker-user", "x-spinnaker-account"];
```

Comment:
It looks like this list has expanded (my fault for telling you `account` when it should be `accounts` also). You can probably just reference these constants instead of redeclaring:

```
  public static final String SPINNAKER_USER = "X-SPINNAKER-USER";
  public static final String SPINNAKER_ACCOUNTS = "X-SPINNAKER-ACCOUNTS";
  public static final String SPINNAKER_USER_ORIGIN = "X-SPINNAKER-USER-ORIGIN";
  public static final String SPINNAKER_REQUEST_ID = "X-SPINNAKER-REQUEST-ID";
  public static final String SPINNAKER_EXECUTION_ID = "X-SPINNAKER-EXECUTION-ID";
```

Source: https://github.com/spinnaker/kork/blob/master/kork-security/src/main/java/com/netflix/spinnaker/security/AuthenticatedRequest.java#L29-L33